css/product.css.map

```diff
@@ -0,0 +1 @@
AAA;UAAA,sCAAA;ACcF;ADXC;EA3XA,gBA4XmB;EA3XnB,iBA2XwB;EA1XxB,mBA0XgC;EAzXhC,6BAyXwC;EAxXxC,mBAwXgE;ACiBjE;ADdC;EA/XA,gBAgYmB;EA/XnB,eA+XwB;EA9XxB,mBA8X8B;EA7X9B,4BA6XsC;EA5XtC,gBA4X6D;ACoB9D;;ADhBA,YAAA;AAEC;EACC,mBAAA;ACkBF;ADhBE;EA3ZD,WA4ZqB;EA3ZrB,YA2Z2B;ACmB5B;ADfC;;EA9YA,gBAgZmB;EA/YnB,eA+YwB;EA9YxB,mBA8Y8B;EA7Y9B,6BA6YsC;EA5YtC,qBA4Y8D;ACqB/D;ADlBC;EAnZA,gBAoZmB;EAnZnB,iBAmZwB;EAlZxB,mBAkZgC;EAjZhC,4BAiZwC;EAhZxC,qBAgZ+D;ACwBhE;ADtBE;EAHD;IAIE,eAAA;ECyBD;AACF;ADtBC;EACC,gBAAA;ACwBF;ADtBE;EAHD;IAIE,gBAAA;ECyBD;AACF;;ADrBA;EApaC,gBAqakB;EApalB,iBAoauB;EAnavB,mBAma+B;EAla/B,4BAkauC;EAjavC,gBAia8D;EAC9D,mBAAA;EACA,qBAAA;EACA,eAAA;EACA,gBAAA;AC4BD;AD1BC;EAPD;IAQE,iBAAA;EC6BA;AACF;;AD1BA,WAAA;AAEC;EACC,mBAAA;AC4BF;AD1BE;EAvcD,WAwcqB;EAvcrB,YAuc2B;AC6B5B;ADzBC;EA1bA,gBA2bmB;EA1bnB,iBA0bwB;EAzbxB,mBAybgC;EAxbhC,WAwbwC;EAvbxC,mBAub8C;AC+B/C;AD5BC;EA9bA,gBA+bmB;EA9bnB,iBA8bwB;EA7bxB,mBA6bgC;EA5bhC,4BA4bwC;EA3bxC,mBA2b+D;ACkChE;AD/BC;EACC,kBAAA;EACA,mBAAA;ACiCF;;AD7BA,oBAAA;AAEC;EA5dA,aA6doB;EA5dpB,cA4d4B;ACgC7B;AD7BC;EACC,UAAA;EAjeD,WAkeoB;EAjepB,YAie0B;ACgC3B;AD9BE;EACC,uCAAA;ACgCH;AD9BG;EAGC,6BAAA;ACgCJ;AD5BE;EACC,kCAAA;AC8BH;AD1BC;EACC,kBAAA;EACA,eAAA;EACA,MAAA;EACA,OAAA;EACA,QAAA;EACA,SAAA;EACA,yBAAA;EAEA,gBAAA;EACA,cAAA;EACA,eAAA;AC4BF;AD1BE;EACC,kBAAA;EACA,WAAA;EAlgBF,WAmgBqB;EAlgBrB,YAkgB2B;EACzB,YAAA;EACA,cAAA;EACA,yBAAA;EAEA,gBAAA;AC6BH;AD1BE;EACC,qBAAA;AC4BH;AD1BG;EACC,kBAAA;AC4BJ;ADvBC;EACC,eAAA;ACyBF;;ADpBA;EACC,mBAAA;ACuBD;ADrBC;EACC,4BAAA;ACuBF;ADrBE;EACC,qBAAA;ACuBH;ADpBE;EACC,gBAAA;ACsBH;ADlBC;EACC,6BAAA;ACoBF;;ADdC;EA9hBA,gBA+hBmB;EA9hBnB,iBA8hBwB;EA7hBxB,mBA6hBgC;EA5hBhC,6BA4hBwC;EA3hBxC,mBA2hBgE;ACqBjE;;ADhBA;EACC,kBAAA;ACmBD;ADjBC;EAviBA,gBAwiBmB;EAviBnB,iBAuiBwB;EAtiBxB,iBAsiBgC;EAriBhC,6BAqiBsC;EApiBtC,qBAoiB8D;ACuB/D;;ADlBA;EACC,6BAAA;EACA,qBAAA;EACA,eAAA;EACA,oCAAA;EACA,oBAAA;ACqBD;ADnBC;EACC,mCAAA;ACqBF;ADlBC;EACC,qBAAA;ACoBF;ADjBC;EACC,WAAA;ACmBF;ADhBC;EAhkBA,gBAikBmB;EAhkBnB,iBAgkBwB;EA/jBxB,mBA+jBgC;EA9jBhC,6BA8jBwC;EA7jBxC,qBA6jBgE;ACsBjE;ADnBC;EApkBA,gBAqkBmB;EApkBnB,iBAokBwB;EAnkBxB,mBAmkBgC;EAlkBhC,4BAkkBwC;EAjkBxC,gBAikB+D;ACyBhE;ADtBC;EACC,mBAAA;ACwBF;;ADnBA;;EAEC,aAAA;EACA,kBAAA;EACA,eAAA;ACsBD;;ADnBA;;EArlBC,gBAulBkB;EAtlBlB,iBAslBuB;EArlBvB,mBAqlB+B;EAplB/B,6BAolBuC;EAnlBvC,gBAmlB+D;EAC/D,kBAAA;EACA,kBAAA;EACA,eAAA;EACA,cAAA;AC0BD;ADxBC;EARD;;IASE,iBAAA;EC4BA;AACF;;ADzBA;;EAEC,WAAA;EACA,kBAAA;EACA,OAAA;EACA,MAAA;EAznBA,aA0nBmB;EAznBnB,cAynB2B;EAC3B,0CAAA;EACA,mBAAA;EACA,gBAAA;AC6BD;;AD1BA;EACC,0CAAA;AC6BD;;AD1BA;;EAEC,WAAA;EAtoBA,aAuoBmB;EAtoBnB,cAsoB2B;EAC3B,iCAAA;EACA,kBAAA;EACA,WAAA;EACA,YAAA;EACA,mBAAA;EAEA,yBAAA;AC8BD;;AD3BA;EACC,UAAA;EAEA,mBAAA;AC8BD;;AD3BA;EACC,UAAA;EAEA,mBAAA;AC8BD;;AD1BA;EACC;IACC,UAAA;EC6BA;AACF;;ADhCA;EACC;IACC,UAAA;EC6BA;AACF;AD1BA;EACC;IACC,qBAAA;EC4BA;AACF;AD/BA;EACC;IACC,qBAAA;EC4BA;AACF;ADzBA;EACC;IACC,UAAA;EC2BA;AACF;AD9BA;EACC;IACC,UAAA;EC2BA;AACF;ADxBA;EACC;IACC,yBAAA;EC0BA;EDxBD;IACC,uBAAA;EC0BA;AACF;ADhCA;EACC;IACC,yBAAA;EC0BA;EDxBD;IACC,uBAAA;EC0BA;AACF","file":"product.css"}
```

Comment:
Okay, what is this?

Reply:
Turns out it can be disabled by adding this in the settings of the Live SASS compiler extension: 	"liveSassCompile.settings.generateMap": false